### DISCUSSION TITLE
Redirect to test servers

### DISCUSSION THREAD

**Discussion by UglyBob79** [persandstrom/python-verisure#168] [Q&A]
Hi!

I'm actually work at Verisure as a consultant and I use their system daily, but in test environments. I also love home automation and want to integrate my alarm to home assistant, but because the servers are basically hard-coded in this library, I can't do it unless I re-write the source. I could ofc do that, but then I wont get updates as easily. Do you see any way to fool this library to use other servers? Or make it possible to config somehow?

**Comment by persandstrom** [comment]
Hi,

Could you add a row to /etc/host and resolve the request for the automation URLs to the IP of your test servers?

**Reply by UglyBob79** [reply]
Ah, would that work? As I run home assistant in docker, I have tried using extra_hosts in the docker compose, to redirect the server names to the test ip:s, but that did not work.

**Reply by persandstrom** [reply]
I don't know :)
